docs/automations/extract-bounding-box-metadata.md

```diff
@@ -0,0 +1,141 @@
+| [Object Type Count](#object-type-count)                                           | Count the types of objects in the group according to label field                                     | Bounding Box Group Property |
+| [Combined Area Ratio](#combined-area-ratio)                                       | Sum of the bounding box size ratio in the group (sum (bounding box area/Image area))                                                           | Bounding Box Group Property |
+| [Bbox Intersection](#bbox-intersection)                                            | Sum of the IOU of all bounding box pairs in the group                                                | Bounding Box Group Property |
+| [Entropy Of Distribution](#entropy-of-distribution)                               | Entropy of the distribution of bounding boxes within a 3 by 3 grid                                   | Bounding Box Group Property |
+| [Mean Aspect Ratio](#mean-aspect-ratio)                                           | The mean of the aspect ratios of a group of bounding boxes                                           | Bounding Box Group Property |
+| [Mean Distance Between Bboxes](#mean-distance-between-bboxes)                     | The mean of the Euclidean distance between each pair of bounding box in the group                    | Bounding Box Group Property |
+| [Mean Relative Area](#mean-relative-area)                                         | The mean of the relative area of bounding boxes in the group                                         | Bounding Box Group Property |
+| [Mean Relative Distance To Image Centre](#mean-relative-distance-to-image-centre) | The mean of the relative distance between each of the bounding box and the image center in the group | Bounding Box Group Property |
+| [Std Aspect Ratio](#std-aspect-ratio)                                             | The standard deviation of the aspect ratios of a group of bounding boxes                             | Bounding Box Group Property |
+| [Std Distance Between Bboxes](#std-distance-between-bboxes)                      | The standard deviation  of the Euclidean distance between each pair of bounding box in the group     | Bounding Box Group Property |
+| [Std Relative Area](#std-relative-area)                                          | The standard deviation of the relative area of bounding boxes in the group                                         | Bounding Box Group Property |
+| [Std Relative Distance To Image Centre](#std-relative-distance-to-image-centre)  | The standard deviation of the relative distance between each of the bounding box and the image center in the group | Bounding Box Group Property |
+
+## Feature Descriptions
+
+### Aspect Ratio
+
+**Aspect ratio** measures the ratio of the bounding box width to its height. It can be useful in scenarios
```

Comment:
We should be consistent with how we capitalize things in title vs in the blurb. I'm going with title casing in the blurb, but would also be okay with just normal capitalization -- we just need to be consistent across each metric.

```suggestion
**Aspect Ratio** measures the ratio of the bounding box width to its height. It can be useful in scenarios
```